web/app/reset-new/page.tsx

```diff
@@ -5,7 +5,7 @@ import ResetBackground from "@/components/Authentication/ResetBackground";
 const Reset: React.FC = () => {
   return (
     <div>
-      <ResetBackground description="Reset password" Card={ResetNew} />
+      <ResetBackground description="Reset password" card={ResetNew} action={undefined} />
```

Comment:
I think it's best we pass a void function (until it's implemented) to the ResetBackground as a prop instead of undefined.

Reply:
Changes have been made